solutions/01_bauerc.txt

```diff
@@ -1,2 +1,2 @@
-Answer: 55123
```

Comment:
these should be exactly what we entered into the textbox on website.

I know later there will be annoying question that printout a bitmap to your terminal with `# and .` we will just have to look at them...